### PR TITLE
Context Manager rollback/commit on aexit

### PR DESCRIPTION
**Pull request by julianit**
According to pyodbc when Connection Manager is used the current transaction should be 
rollback-ed on error, otherwise commit-ed if autocommit=False

### CODE REVIEW

File: tests/test_pool.py [hunk 1]
```diff
@@ -1,454 +1,490 @@
...
-    assert cur.closed
+import asyncio
+
+import pytest
+import aioodbc
+from aioodbc import Pool, Connection
+from pyodbc import Error
+
+@pytest.mark.asyncio
+async def test_create_pool(loop, pool_maker, dsn):
```
Comment (jettify) [inline tests/test_pool.py]: Odd, why github shows diff here?

**Comment by jettify** [comment]
Changes looks good! Need to investigate why github shows diff there

File: tests/test_pool.py (hunk 1 above)
Comment (jettify) [reply tests/test_pool.py]: ah, good point, can you please convert to unix end of line?

Comment (julianit) [reply tests/test_pool.py]: I updated the file but still the same issue. Do you have any other ideas?

**Comment by jettify** [comment]
looks good now! thank!

**Comment by jettify** [comment]
New version deployed on PyPI  https://pypi.org/project/aioodbc/0.3.2/

**Pull request merged**